src/form_parser/formparser/html/required_fields.py

```diff
@@ -0,0 +1,269 @@
+# -*- coding: utf-8 -*-
+"""
+Rúbia Reis Guerra
+rubia-rg@github
+Find required fields in form
+"""
+import logging
+
+from formparser import utils
+from multiprocessing import Pool, cpu_count
+from pyppeteer import launch
+from syncer import sync
+
+
+class RequiredFields:
+    """Identify required fields in web forms"""
+
+    def __init__(self, url: str, fillers):
+        """Constructor for RequiredFields
+
+        Args:
+            url (`str`): target url.
+            fillers (`dict`): dictionary of field types and values use as
+            fillers for testing
+        """
+        self.url = url
+        self.required_fields = list()
+        self.fillers = fillers
+
+    @sync
+    async def get_page(self):
+        """Wrapper for async get page"""
+        browser = await launch(headless=True)
```

Comment:
```suggestion
        browser = await launch(headless=True, dumpio=True)
```
Incluir esse argumento opcional como True ajuda a evitar alguns bugs do Pyppeteer, acho que pode ajudar na estabilidade do módulo!